debezium-server-http/src/test/java/io/debezium/server/http/standard_webhooks/StandardWebhooksAuthenticatorTest.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.server.http.standard_webhooks;
+
+import static org.mockito.Mockito.mockStatic;
+import static org.mockito.Mockito.withSettings;
+
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.net.http.HttpHeaders;
+import java.net.http.HttpRequest;
+import java.time.Instant;
+import java.util.Optional;
+import java.util.UUID;
+
+import org.junit.Test;
+import org.junit.jupiter.api.Assertions;
+import org.mockito.MockedStatic;
+
+public class StandardWebhooksAuthenticatorTest {
+
+    @Test
+    public void addAuthorizationHeader() throws URISyntaxException {
+        UUID messageId = UUID.fromString("22bd292a-71ab-46fe-a460-8632d6754ac6");
+        Instant instant = Instant.ofEpochSecond(1234);
+
+        try (
+                MockedStatic<Instant> mockedInstant = mockStatic(Instant.class,
```

Comment:
You can avoid `mockStatic` using the `Clock` approach described in my previous comment.